README.md

```diff
@@ -59,4 +59,107 @@ From within the root of the repo run the following command to build an image of
 Run `renterd` in the background as a container named `renterd` that exposes its
 API to the host system and the gateway to the world.
 
-`docker run -d --name renterd -e RENTERD_API_PASSWORD="<PASSWORD>" -e RENTERD_WALLET_SEED="<SEED>" -p 127.0.0.1:9980:9980/tcp -p :9981:9981/tcp <IMAGE_ID>`
\ No newline at end of file
+`docker run -d --name renterd -e RENTERD_API_PASSWORD="<PASSWORD>" -e RENTERD_WALLET_SEED="<SEED>" -p 127.0.0.1:9980:9980/tcp -p :9981:9981/tcp <IMAGE_ID>`
+
+# Usage
+
+## Wallet
+
+Make sure the wallet is funded. Fetch the wallet's address and transfer some money. Verify the wallet's balance is not zero using the following endpoints:
+
+- `GET /api/bus/wallet/address`
+- `GET /api/bus/wallet/balance`
+
+The autopilot will automatically redistribute the wallet funds over a certain of outputs that make sense with regards to the autopilot's configuration. Contract formation and renewals work best when the autopilot has a good amount of outputs at its disposal. It's definitely a good idea to verify whether this is the case because if not it means that it's likely the autopilot is misconfigured, in which case the logs should be of help.
+
+- `GET /api/bus/wallet/outputs`
+
+## Consensus
+
+In order for the contracts to get formed, your node has to be synced with the blockchain. If you are not bootstrapping your node this can take a while. Verify your node's consensus state using the following endpoint:
+
+- `GET /api/bus/consensus/state`
+
+## Config
+
+To have a working autopilot, it must be configured with a sane config. The
+autopilot's configuration is configurable through the following endpoints:
+
+- `GET /api/autopilot/config`
+- `PUT /api/autopilot/config`
+
+Especially the `contracts` section is important, make sure the `amount` is set to the amount of hosts with which you want to form a contract. The `allowance` is the amount of money the autopilot can spend per period, make sure it is not set to zero or contracts won't get formed.
+
+```json
+{
+	"wallet": {
+		"defragThreshold": 1000
+	},
+	"hosts": {
+		"ignoreRedundantIPs": false,
+		"scoreOverrides": {}
+	},
+	"contracts": {
+		"set": "autopilot",
+		"amount": 50,
+		"allowance": "10000000000000000000000000000",
+		"period": 6048,
+		"renewWindow": 2016,
+		"download": 1099511627776,
+		"upload": 1099511627776,
+		"storage": 1099511627776
```

Comment:
I'd leave comments for the large numbers to tell the reader what they translate to. e.g. 10TiB or 10KS.